app/views/layouts/components/_footer_scripts.html.haml

```diff
@@ -1 +1,2 @@
+= render 'layouts/components/footer_content'
```

Comment:
this file is for just the footer scripts. the footer content won't be here. the render for the footer content should be part of the layout. it would only be on the landing pages and not on all the other layouts that we have.